AAT_gae/src/main/java/com/aat/restlet/AttendanceResource.java

```diff
@@ -19,17 +19,24 @@
 import com.aat.datastore.Course;
 import com.aat.utils.ResourceUtil;
 
+import java.util.List;
+
 public class AttendanceResource extends ServerResource {
 	
 	@Post
 	public Representation create(Representation entity) {
+		// Check if of type Student
+		ResourceUtil.checkTokenPermissions(this, Student.class);
```

Comment:
I am not sure, there are only certain methods that need to check the token permissions. Also, since certain resources are only for students and certain are only for tutors, that may make it tricky. Do you have any suggestions on how that would be done?